From de73f7c09dd2b5c9aa091ffd4bd5df89d97c23b8 Mon Sep 17 00:00:00 2001
From: albf <albf.unicamp@gmail.com>
Date: Mon, 16 Apr 2018 02:21:39 -0400
Subject: [PATCH] Use stack from java.util

---
 Expression.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Expression.java b/Expression.java
index 08ba3ed..e499f7a 100644
--- a/Expression.java
+++ b/Expression.java
@@ -2,6 +2,7 @@ package rExp4j;
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.Stack;
 
 public class Expression {
     private final List<Double> tokens;
@@ -11,7 +12,7 @@ public class Expression {
     }
 
     public double Evaluate() {
-        ArrayStack arrayStack = new ArrayStack();
+        Stack<Double> arrayStack = new Stack<>();
 
         this.tokens.forEach(x -> {
             if (arrayStack.isEmpty()) {
-- 
2.11.0

