From c5bcc6ebe3c608856d19fb500509932eee5045bf Mon Sep 17 00:00:00 2001
From: albf <albf.unicamp@gmail.com>
Date: Tue, 17 Apr 2018 18:56:04 -0400
Subject: [PATCH] use-unsafe-math-operations

---
 IntOperations.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/IntOperations.java b/IntOperations.java
index 756b614..6b35b84 100644
--- a/IntOperations.java
+++ b/IntOperations.java
@@ -2,9 +2,9 @@ package rJacop;
 
 public class IntOperations {
     int result(int w, int x, int y, int z) {
-        int resultA = Operations.multiplyInt(w, x);
-        int resultB = Operations.addInt(y, z);
-        return Operations.subtractInt(resultA, resultB);
+        int resultA = w * x;
+        int resultB = y + z;
+        return resultA - resultB;
     }
 }
 
-- 
2.11.0

